docs/wiki/Fetching latest changes - Scenario 2 Module library and CI environment.md

```diff
@@ -0,0 +1,55 @@
+In this scenario you have adopted the [Pipeline Orchestration](./Solution%20creation.md#pipeline-orchestration) approach and you have onboarded the module library and the CI environment, as described in [Getting started Scenario 2 - Onboard module library and CI environment](./Getting%20started%20-%20Scenario%201%20Onboard%20module%20library%20and%20CI%20environment.md).
+
+Depending on the DevOps environment you have chosen (GiutHub or DevOps) you may have a different situation.
+
+### _Navigation_
+
+- [GitHub public repository](#lgithub-public-repository)
+- [GitHub private repository](#github-private-repository)
+- [Azure DevOps private git](#azure-devops-private-git)
+
+# GitHub public repository
+You have a public fork of public CARML source repository in your target organization.
+The update process is the following:
+1. Keep your fork synced to the fork upstream repository, on the GitHub web UI or through the GitHub CLI or the command line, as explaind in [Syncing a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) documentation.
+1. Sync your local copy of the fork taking care of eventual customizations you can have in place.
+1. Personalize files with your specific settings:
+    1. [Update default name prefix](./Getting%20started%20-%20Scenario%202%20Onboard%20module%20library%20and%20CI%20environment.md#31-update-default-nameprefix)
```

Comment:
Unless you reference a ReadMe, please don't use the `.md` extension in the link, as it won't work in the Wiki

Reply:
Note: Please only resolve this comment after removing the `.md` references ;)